### ISSUE TITLE
copy and pasted the model_web. I got this error.

### ISSUE THREAD

**Issue by akkadian-picseal** [cloud-annotations/object-detection-live-stream#2]
models@0.1.1:1 Uncaught (in promise) Error: Tensor must have a shape comprised of positive integers but got shape [1917,].
    at z (models@0.1.1:1)
    at models@0.1.1:1
    at Array.forEach (<anonymous>)
    at it (models@0.1.1:1)
    at Tn (models@0.1.1:1)
    at _n (models@0.1.1:1)
    at models@0.1.1:1
    at models@0.1.1:1
    at Kt.scopedRun (models@0.1.1:1)
    at Kt.tidy (models@0.1.1:1)

----------------------------------------------------------

I tried  other samples on your git.

I tried https://github.com/tensorflow/tfjs-models/tree/master/coco-ssd with the model.
It was not succeed also. Help me out please.

**Comment by bourdakos1** [comment]
I think you don't have enough training data or not enough labels, try to have at least 2 labels and 20 images per label. Otherwise this error pops-up somewhat randomly

**Comment by akkadian-picseal** [comment]
I have 6 labels and used 200 images.
I got my model trained with IBM cloud.  cacli ver 1.3.2 (latest one)  I got label, model.json and 6 bin files.

**Comment by bourdakos1** [comment]
hmm yea that's definitely enough training data... Do you mind emailing me a screenshot of your training data and maybe share your bucket with me? bourdakos1@gmail.com

**Comment by akkadian-picseal** [comment]
Sorry for the late response. I am living in other time zone.
I email you with bucket and files.
